### ISSUE TITLE
"Failure details copied" toast message is not announced by Narrator on Edge

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#4283]
## Describe the bug

```aria-live="polite"``` is present on the toast message that appears when ```Copy failure details``` button is selected in the ```More options``` menu on an instance card. But, narrator does not announce this message on the Edge extension.

**To Reproduce**

1. Turn on Narrator
2.  Navigate to https://www.washington.edu/accesscomputing/AU/before.html on Edge
2. Run Automated checks
2. Using keyboard, expand the ```more options``` button on instance card in details view 
3. Select the ```copy failure details``` option
4. Observe that the toast message (Failure details copied) appears but Narrator does not announce it.

## Expected behavior

The ```Failure details copied``` toast message should be announce by all the screen readers on both Edge and Chrome extensions

## Screenshots
![image](https://user-images.githubusercontent.com/22068434/119572251-4c0af600-bd67-11eb-8fdb-987631568ed1.png)

## Context

- OS Name & Version: Windows 10 21H1 Build 19043.985
- AI-Web Version & Environment: Version 2.26.1 Prod
- Browser Version: Microsoft Edge Version 90.0.818.66 (Official build) (64-bit)
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

**Comment by karanbirsingh** [comment]
In Edge I can reproduce this even with `aria-live` set to `assertive`. Focus moves to the closed menu and the Toast is not mentioned. In Chrome Narrator announces the Toast. In both browsers the Toast is announced in the target page "copy failure details" button.

I ran AI-Windows against the Cards UI and target page copy-failure-details button. In both cases the StructureChanged events (ChildAdded) seemed identical.

I am trying to put together a repro in Codepen, but no luck yet (the Toast is announced): https://codepen.io/karansin/pen/eYvEvEM

**Comment by karanbirsingh** [comment]
This repros in Narrator + Edge when the `aria-live` element is a descendent of an `li` element. Here is a minimal repro: https://codepen.io/karansin/pen/OJpjKgV

Our cards are list items, so it affects us. Will reach out to see if I can report this

**Comment by iamrafan** [comment]
@karanbirsingh I shared the minimal repro with the TT team and asked them to file a bug on Narrator

**Comment by peterdur** [comment]
Verified the notification "Failure Details copied" is read by Narrator on Edge 105.
